dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -348,7 +368,19 @@ else if (Guard.isNullOrEmpty(values)) {
         }
     }
 
-    private void refreshBlackPearlSideItem(final TreeItem<Ds3TreeTableValue> treeItem) {
+    private static boolean isEmptyFolder(final Path path) {
+        if (Files.isDirectory(path)) {
```

Comment:
If you reverse this if statement, and do `if (!Files.isDirectory(path)) { return false; }`
then you can reduce the block indentation for the if-true case